src/hardware/configuration.rs

```diff
@@ -825,6 +825,14 @@ pub fn setup(
         None
     };
 
+    let cycle_counter = {
+        // Set TRCENA bit, which is required for DWT counter to tick. (This is also
+        // set automatically when running in the debugger, and only cleared on power
+        // reset, not on soft reset.)
+        core.DCB.enable_trace();
```

Comment:
This isn't an issue with other platforms because RTIC, when using the CycleCounter as a monotonic, automatically generates the `core.DCB.enable_trace()` call in the APP macro.